src/atcore.cpp

```diff
@@ -101,9 +101,12 @@ void AtCore::findFirmware(const QByteArray &message)
         if (file.startsWith(QStringLiteral("lib"))) {
             file = file.remove(QStringLiteral("lib"));
         }
+        QByteArray fwName = message;
+        fwName.remove(0, 14); //remove FIRMWARE_NAME:
```

Comment:
This form isn't robust. Maybe a combination of contains, split and other things.